src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Client/SolutionExplorer/ProjectNodeCommandHandler.cs

```diff
@@ -100,7 +100,8 @@ private void OpenFile(WorkspaceVisualNodeBase node)
                 {
                     Assumes.NotNull(openDocumentService);
 
-                    await openDocumentService.OpenDocumentAsync(node.NodeMoniker, cancellationToken: default);
+                    if (openDocumentService != null)
+                        await openDocumentService.OpenDocumentAsync(node.NodeMoniker, cancellationToken: default);
```

Comment:
Maybe add an empty `else` clause with a `// TODO: figure out what to tell the user if we can't get the service.`.

Reply:
Added it. Also created a tracking issue: https://github.com/dotnet/project-system/issues/6306

Reply:
Can you add the a link to the github issue in the comments? 

---
In reply to: [445717302](https://github.com/dotnet/project-system/pull/6305#discussion_r445717302) [](ancestors = 445717302)

Reply:
Sure, will add it and merge the pr